jmix-flowui/flowui/src/main/resources/io/jmix/flowui/view/layout.xsd

```diff
@@ -3950,6 +3950,18 @@
             <xs:enumeration value="CASE_INSENSITIVE"/>
         </xs:restriction>
     </xs:simpleType>
+
+    <!-- NavigationMenuBar -->
+    <xs:complexType name="navigationMenuBarComponent">
+        <xs:complexContent>
+            <xs:extension base="baseComponent">
+                <xs:attribute name="loadMenuConfig" type="xs:boolean"/>
+
+                <xs:attributeGroup ref="hasSize"/>
+                <xs:attributeGroup ref="hasClassNames"/>
```

Comment:
Add `css` attribute

Reply:
`css` exists in parent `baseComponent` type:
```

    <!-- Utils -->
    <xs:complexType name="baseComponent">
        <xs:attributeGroup ref="hasId"/>
        <xs:attribute name="visible" type="xs:boolean"/>
        <xs:attribute name="colspan" type="xs:integer"/>
        <xs:attribute name="alignSelf" type="alignEnum"/>
        <xs:attribute name="css" type="xs:string"/>

        <!-- enable XSD extension here -->
        <xs:anyAttribute namespace="##other" processContents="lax"/>
    </xs:complexType>
```